RELEASES.md

```diff
@@ -1,3 +1,122 @@
+Version 1.43.0 (2020-04-23)
+==========================
+
+Language
+--------
+- [Fixed using binary operations with `&{number}` (e.g. `&1.0`) not having
+  the correctly inferred type.][68129]
+
+**Syntax only changes**
+- [Allow `type Foo: Ord` syntactically.][69361]
+- [Unify item parsing & filter illegal item kinds.][69366]
```

Comment:
This PR lists "RELNOTES: Now, item macro fragments can be interpolated into impl, trait, and extern contexts ..." as what to put here -- and that sounds like a non-syntax-only change.

Reply:
That's still a syntax-only change. It's the general item parser that will now accept `NtItem` and the foreign/assoc item parsers will call the general item parser and transform them. I would include an example here cause its an enabling change for macro authors.

Reply:
(You'll need to fix the suggestion manually... GitHub is being daft wrt. the backticks...)

```suggestion
- [`item` macro fragments can be interpolated into `trait`s, `impl`s, and `extern` blocks.][69366]

  For example, you may now write:
  ```rust 
  macro_rules! mac_trait {
      ($i:item) => {
          trait T { $i }
      }
  }

  mac_trait! {
      fn foo() {}
  }
  ```
```